src/main/java/gregtech/loaders/recipe/handlers/PartsRecipeHandler.java

```diff
@@ -549,6 +594,15 @@ public static void processTurbine(OrePrefix toolPrefix, Material material, Ingot
                     'P', new UnificationEntry(OrePrefix.plateDouble, material),
                     'S', new UnificationEntry(OrePrefix.screw, material));
         }
+
+        if (material.hasFluid() && material.getProperty(PropertyKey.FLUID).solidifiesFrom() != null) {
```

Comment:
I feel like we should keep the turbine rotors to forming press only, as it gives some use to an underused machine.